_posts/2017/2017-30-03-elixir-meetup-at-epages-what-is-the-best-vindinium-bot.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Elixir Meeetup at ePages: what's the best vindinium bot?"
+date: "2017-03-30 06:39:20"
+image: blog-header/elixir.jpg
+categories: events
+authors: ["Rose"]
+---
+<style>
+.twitter-tweet {
+  margin: auto;
+}
+</style>
+
+This Tuesday, the second part of the [hh.ex vindinium challenge](https://www.meetup.com/de-DE/Elixir-Hamburg/events/238421135/?eventId=238421135) took place in the [ePages](https://www.epages.com/en/) office in Hamburg.
+Despite the amazing weather conditions outside, there were quite a few [Elixir](http://elixir-lang.org/) and [vindinium](http://vindinium.org/) interested developers to join the meetup.
+
+## Programming a semi static blog engine with Elixir
+
+The event started off with a presentation about a side project of our developer Erik.
+He presented his semi static blog engine "Jelly Shot".
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/Air2ick">@Air2ick</a> presents his semi static blogengine <a href="https://twitter.com/hashtag/JellyShot?src=hash">#JellyShot</a> with <a href="https://twitter.com/elixirlang">@elixirlang</a> <a href="https://twitter.com/HamburgEx">@HamburgEx</a> <a href="https://twitter.com/hashtag/meetup?src=hash">#meetup</a> <a href="https://twitter.com/hashtag/hamburg?src=hash">#hamburg</a> cc/ <a href="https://twitter.com/epagesdevs">@epagesdevs</a></p>&mdash; harm (@netzfisch) <a href="https://twitter.com/netzfisch/status/846769010389798912">March 28, 2017</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+The idea to set up this blog engine with [Elixir](http://elixir-lang.org/) came up, because for him, working with the static site generator [Jekyll](https://jekyllrb.com/) often takes up too much time.
+Motivated to immerse in the world of [Elixir](http://elixir-lang.org/), he thought it couldn't be too difficult to write his own blog engine.
+And he was absolutely right!
+
+His blogengine Jelly Shot uses the web-framework [Phoenix](http://www.phoenixframework.org/) and is written in [Elixir](http://elixir-lang.org/).
```

Comment:
Mentioned the 3rd time that this thing is writte nin elixir, would remove this here...